spectrum/test_article.py

```diff
@@ -130,6 +129,32 @@ def test_article_with_unicode_content(generate_article):
     journal_page = checks.JOURNAL.article(id=article.id(), has_figures=article.has_figures())
     assert "Szymon \xc5\x81\xc4\x99ski" in journal_page
 
+@pytest.mark.journal
+def test_googlebot_sees_citation_metadata(generate_article):
+    article = generate_article(template_id=SIMPLEST_ARTICLE_ID)
+    _ingest_and_publish(article)
+    checks.API.wait_article(id=article.id())
+
+    page = checks.JOURNAL_GOOGLEBOT.article(id=article.id())
+
+    soup = BeautifulSoup(page, "html.parser")
+    citation_metadata = soup.find_all("meta", {"name": re.compile("^citation_")})
+
+    assert len(citation_metadata) > 0, "Expected citation metadata, found none"
+
+@pytest.mark.journal
+def test_public_does_not_see_citation_metadata(generate_article):
```

Comment:
can probably do both checks with the same test and published article (which also smoke checks we are not caching with the wrong `Vary`), unlike in unit tests the time to run them all builds up fast